elixir/apps/web/lib/web/mailer/auth_email/new_user.text.heex

```diff
@@ -0,0 +1,22 @@
+Welcome to Firezone!
+
+You've been invited to the following Firezone Account:
+
+<%= "#{@account.name}\n" %>
+
+To get started accessing your resources, you will need to install one of the Firezone clients:
+
+https://www.firezone.dev/kb/user-guides/apple-client
+https://www.firezone.dev/kb/user-guides/windows-client
+https://www.firezone.dev/kb/user-guides/android-client
+https://www.firezone.dev/kb/user-guides/linux-client
+
+
+After installing the client, click "Sign In" and when prompted for an account name use the following:
+
+<%= "#{@account.slug}\n" %>
```

Comment:
Why do we have to add `\n` here?

Reply:
In my testing, the formatting never turned out correctly unless I added the `\n` characters in the string itself.

Maybe this was just the swoop mailbox preview not displaying correctly? 🤷‍♂️   I tried adding `<br />` tags in places, but it just seemed to make things even more strange/unexpected, so this is what I ended up with.

Here's the output with the `\n` in the string:
```
Welcome to Firezone!

Firezone Admin (firezone@localhost) invited you to the following Firezone Account:
Firezone Account

To get started accessing your resources, you will need to install one of the Firezone clients:

https://www.firezone.dev/kb/user-guides/apple-client
https://www.firezone.dev/kb/user-guides/windows-client
https://www.firezone.dev/kb/user-guides/android-client
https://www.firezone.dev/kb/user-guides/linux-client


After installing the client, click "Sign In" and when prompted for an account name use the following:
firezone

If you feel this message has been sent to you by mistake, you can safely ignore this email.

Thanks,
The Firezone Team
```

And here's the output without the `\n` in the string:

```
Welcome to Firezone!
Firezone Admin (firezone@localhost) invited you to the following Firezone Account:
Firezone Account
To get started accessing your resources, you will need to install one of the Firezone clients:

https://www.firezone.dev/kb/user-guides/apple-client
https://www.firezone.dev/kb/user-guides/windows-client
https://www.firezone.dev/kb/user-guides/android-client
https://www.firezone.dev/kb/user-guides/linux-client


After installing the client, click "Sign In" and when prompted for an account name use the following:
firezone
If you feel this message has been sent to you by mistake, you can safely ignore this email.

Thanks,
The Firezone Team
```

Reply:
What do you think about making them more explicit by doing something like `<%= "\n" %>`?

Reply:
I'll give it a try.